### PR TITLE
crudely created a solution that isn't tested yet

### PR DESCRIPTION
**Pull request by WovenCoast**
replicating [this section of code](https://github.com/abh80/smart-edge/blob/master/app/src/main/java/com/abh80/smartedge/plugins/MediaSession/MediaCallback.java#L16), crudely implemented a fix for [App crashes when phone call is picked up#34](https://github.com/abh80/smart-edge/issues/34)

### CODE REVIEW

**Comment by abh80** [comment]
Will try testing it

**Review by GithubBoat — APPROVED** [review]
looks no harm lets try it

**Comment by GithubBoat** [comment]
@WovenCoast,Great work!, looks good to merge, once ready post a comment saying `ready to merge`

**Comment by WovenCoast** [comment]
thanks, it's ready to merge

**Pull request merged**
